frontend/packages/enduser-frontend/src/localization/app/fi.json

```
@@ -1005,6 +1005,9 @@
         "Sallitut tiedostomuodot:",
         "PDF, JPEG/JPG, PNG ja DOC/DOCX"
       ]
-    }
+    },
+    "modal-header": "File processing is ongoing",
```

Nit: "in progress" is better in this context. "Ongoing" is usually used to refer to something longer and more constant.
```suggestion
    "modal-header": "File processing is in progress",
```